CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-﻿# 0.7.0
+﻿# 0.7.1
+* The Results type now allows unlimited results by setting pageSize to -1.
```

Comment:
It seems like it actually allows it with any pageSize < 1, including 0 and any negative number, right?

Reply:
true! changing it